app/src/main/java/com/hover/stax/fieldstates/FieldStateType.java

```diff
@@ -0,0 +1,6 @@
+package com.hover.stax.fieldstates;
+
+public enum FieldStateType {
```

Comment:
I thought of it that, enum was a better use here cos:

- It houses the types of field state we have and helps in code readability. Instead of passing integers that could accept invalid data.